Trace.py

```diff
@@ -314,7 +314,16 @@ def _dump_texture(self, index):
             return ""
 
         offset = self.xbox.read_u32(XboxHelper.PGRAPH_TEXOFFSET0 + reg_offset)
-        pitch = self.xbox.read_u32(XboxHelper.PGRAPH_TEXCTL1_0 + reg_offset) >> 16
+        # FIXME: read pitch from registers instead of guessing
+        # FIXME: clean up associated fallback code in Texture.py
+        # There is a case in Morrowind where pitch is set to 0x8 which is clearly wrong
+        #   Texture 0 [0x03CC6000, 128 x 128 x 1 (pitch: 0x8), format 0xC]
+        #   Texture 1 [0x03C98000, 64 x 64 x 1 (pitch: 0x8), format 0x6]
+        # Double check the register for correctness and see if swizzling has something
+        # to do with it (e.g., like how NV097_SET_TEXTURE_IMAGE_RECT is only needed by
+        # linear formats).
```

Comment:
Not sure where the comment originated (is it mine?).
But pitch is only used for unswizzled formats - and it *is* used there.